### ISSUE TITLE
Studio opens a "false" file explorer.

### ISSUE THREAD

**Issue by mrfiguy** [vinegarhq/vinegarhq.github.io#128]
When I try to open a place file from files, it opens a "false" explorer with nothing from my actual drive.
![ksnip_20240723-220055](https://github.com/user-attachments/assets/8fd0dbed-a7cc-4f1e-8d53-0094dc9d6074)
(I've looked through every folder in the "false" drive and tried changing the default directory in wine settings, alongside trying to find where studio is getting this based off of the l.rbxl file.)

**Comment by jrelvas-ipc** [comment]
Yeah - this really doesn't seem possible to fix unless Wine ever gets native file picker functionality.
